test/test_helper.rb

```diff
@@ -1,9 +1,10 @@
 require 'coveralls'
+require 'simplecov'
 require 'minitest/autorun'
 require 'webmock/minitest'
 require 'razorpay'
 
-Coveralls.wear!
+Coveralls.wear! if ENV['RUBY_ENV'] == 'travis'
```

Comment:
Change this check as well. I'm concerned because coveralls didn't trigger for this PR